.github/workflows/ci.yml

```diff
@@ -44,4 +44,4 @@ jobs:
         run: make submodules
 
       - name: 🌌 Coq Universe
-        run: opam exec -- make universe
+        run: opam exec -- make universe DUNEOPT=""
```

Comment:
We can remove `--error-reporting=twice` from the default and put it here instead.

Reply:
If we build with quiet, do we need that setting?

Reply:
Warnings will get printed in the quiet build, but errors will be hard to find.